fdbserver/TenantBalancer.actor.cpp

```diff
@@ -1255,7 +1268,7 @@ ACTOR Future<Void> abortMovementDueToFailedDr(TenantBalancer* self, MovementReco
 
 	wait(abortPeer(self, *record));
 
-	record->movementState = MovementState::ERROR;
+	record->setMovementError("Abort movement due to failed DR.");
```

Comment:
I think we may want to incorporate more details here once we are actually reading DR status, but I think we could have some text like this for now:

```suggestion
	record->setMovementError("The DR copying the data has failed or stopped running");
```